app/helpers/glossary_term_helper.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# View Helpers for Glossary
+module GlossaryTermHelper
+  def destroy_button(term)
```

Comment:
I've run into this before, and I'm realizing that I'm increasingly in the minority here, but I would like to officially at least suggest that we depart from the standard, and change the routing to allow GET requests to destroy.  There are several problems with requiring DELETE.  One, you have to kludge the styling of the "buttons".  Two, it requires JS.  Three, you cannot right-click the button like you can a normal link.  It adds up.  To me, the whole thing reeks of "this is not the 'right' way to do it".  We have to try to hard to get it to work.  So much easier to accept GET instead.